test/e2e/openshift/openshift_test.go

```diff
@@ -85,6 +85,62 @@ var _ = Describe("Azure Container Cluster using the OpenShift Orchestrator", fun
 		}
 	})
 
+	It("should whitelist pods on compute nodes", func() {
```

Comment:
something like "should deploy all pods onto appropriate nodes" would be clearer?

Reply:
Currently we are only checking `compute` node types.  Should we add master and infra types as well?